styles/stylesheet.css

```diff
@@ -0,0 +1,303 @@
+main {
+	padding: 1em 2em 1em 2em;
+	/* top, right, bottom, left */
+}
+
+.info {
+	border: 1.5px solid rgb(220, 220, 220);
+	padding: 0;
+	margin-left: 0;
+	margin-bottom: 1em;
+}
+
+
+/*print needs more work*/
+
+@media print {
+	header,
+	nav,
+	footer,
+	.preview {
+		display: none;
+	}
+	.collg4 {
+		width: 33%;
+	}
+}
+
+@media screen and (max-width: 1366px) and (orientation: landscape),
+(max-width: 736px) and (orientation: portrait) {
+	.preview p
+	/*, .preview img*/
+	/*images too? or just paragraph preview?*/
+	{
+		display: none;
+	}
+}
+
+
+/* media queries for flexbox per assignment 6 commented out - boodstrap used instead*/
+
+
+/*.infoparent {
+	display: flex;
+	flex-flow: row wrap;
+}*/
+
+header {
+	background-image: url("../images/header-image.jpg"), linear-gradient(to left, grey, rgb(25, 60, 93));
+	background-position: top right;
+	background-repeat: no-repeat;
+	background-size: cover;
+}
+
+header nav {
+	font-weight: bolder;
+	font-family: "Arial Black", "Arial Bold", "Ariel";
+	text-align: center;
+	color: rgb(25, 60, 93);
+	background-color: rgb(236, 236, 236);
+}
+
+nav a {
+	font-family: serif;
+	color: rgb(25, 60, 93);
+	padding: 1.5rem;
+	margin: 1.5rem;
+	display: inline;
+}
+
+footer {
+	height: auto;
+	width: 100%;
+	bottom: 0;
+	top: 0;
+	background-color: rgb(61, 61, 61);
+	color: white;
+	text-align: center;
+	margin: auto;
+	padding-left: 8%;
+	padding-right: 8%;
+}
+
+footer nav {
+	padding: 0;
+	text-align: center;
+	color: white;
+	margin: 0;
+	background-color: inherit;
+}
+
+footer nav a {
+	color: white;
+	padding: 1rem;
+	margin: 1rem;
+}
+
+
+/*Small Devices, Tablets */
+
+@media only screen and (max-width: 991px) {
+	/*.colsm12 {width: 100%;}*/
+	header nav {
+		font-size: 1em;
+	}
+}
+
+
+/* Medium Devices, Desktops */
+
+@media only screen and (min-width: 992px) {
+	/*.colmd6 {width: 49.9%;}*/
+	header nav {
+		font-size: 1.5em;
+	}
+}
+
+
+/* Large Devices, Wide Screens */
+
+@media only screen and (min-width: 1200px) {
+	/*.collg4 {width: 33%;}*/
+	header nav {
+		font-size: 2em;
+	}
+}
+
+.col-sm-offset-4 nav {
+	text-align: center;
+	padding: 1em;
+	color: rgb(25, 60, 93);
+	margin: 0;
+	background-color: rgb(220, 220, 220);
+}
+
+header img {
+	padding-top: 1em;
+	padding-left: 4em;
+	max-width: 50%;
+}
+
+footer img {
+	padding-top: 5em;
+	max-width: 50%;
+}
+
+img {
+	width: 100%;
+}
+
+a {
+	color: rgb(81, 156, 183);
+	font-weight: bold;
+	display: block;
+	text-decoration: none;
+}
+
+a:hover,
```

Comment:
I believe this is causing some issues with your menu items "Jitter" on rollover.